### DISCUSSION TITLE
Speed2Fly via NMEA

### DISCUSSION THREAD

**Discussion by sandertje567** [iltis42/XCVario#197] [Q&A] [answered]
Hiy all,
We have our XCVario 22 conected to a Stefly OpenVario (including remote) via the S2 port. Controlling the Mcready and bugs via the OV remote works flawlessly. Hower we are unable to control the Speed2Fly command via the NMEA. We adopted the OV to send “$ g , s 1 *<cs> <CR> <LF>”. Nothing happens on the XCv. The serial monitor on thr XCv shows the sentence when send by the OV.
Does anyone have this working?
Thnx!

**Comment by XCNav** [comment, marked as answer]
yes, we got it running on our XCRemote
We created a new Vario Menu. Here you can switch from Vario to S2F via Softbutton on the Screen or via double click on our remote centerstick. Here is the .xci for the new Vario Menu. We can also adjust Volume by that!


mode=Vario_menu
type=key
data=RIGHT
event=SendNMEAPort2 g,v5
event=SendNMEAPort1 g,v5
event=StatusMessage VOLUME UP
label=VOLUME UP (Right)
location=8

mode=Vario_menu
type=key
data=LEFT
event=SendNMEAPort2 g,v-5
event=SendNMEAPort1 g,v-5
event=StatusMessage VOLUME DOWN
label=VOLUME DWN (Left)
location=9

mode=Vario_menu
type=key
data=0
event=SendNMEAPort2 g,s1
event=SendNMEAPort1 g,s1
event=StatusMessage SOLLFAHRT
label=STF
location=3

mode=Vario_menu
type=key
data=0
event=SendNMEAPort2 g,s0
event=SendNMEAPort1 g,s0
event=StatusMessage Vario
label=Vario
location=2
